src/main/java/com/tracelink/appsec/ariadne/utils/Utils.java

```diff
@@ -18,69 +18,114 @@
 
 import java.util.Arrays;
 import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+/**
+ * Utilities class for formatting and parsing artifact names.
+ */
 public class Utils {
-    ////////////////////////////////////////////// Assume bad formatting ///////////////////////////////////////////////
 
-    public static String getFullName(String artifact) {
-        return String.join(":", formatArtifact(artifact));
-    }
+	private static final Logger LOG = LoggerFactory.getLogger(Utils.class);
+
+	/*
+	 * Assume bad formatting
+	 */
```

Comment:
What does this comment mean?

Reply:
I think I understand. This might belong in the individual methods along with the Assume Good Formatting comment below

Reply:
Added some more detail about assumptions to the Javadoc for each method.